### ISSUE TITLE
Do we really want to maintain our own fonts repository?

### ISSUE THREAD

**Issue by jancborchardt** [opensourcedesign/fonts#21]
We already list 3 big platforms of where to get openly licensed fonts on our [Resources page](https://opensourcedesign.net/resources/#fonts) – Open Font Library, The League of Movable Type, and Google Fonts. A pull request to add further font resources is at https://github.com/opensourcedesign/opensourcedesign.github.io/pull/241

Rather than adding fonts here, needing to maintain them, update outdated fonts (see "Fira out of date #13", "Polish translation for Free fonts needs update #17"), we should encourage people to submit their typefaces to the existing platforms.

And even if they are not on these platforms (e.g. like [Public Sans](https://github.com/uswds/public-sans)), it would be better to link to the original repository on the resources page than put them in our own repository at the danger of them being outdated. (Related, @freder already collected a huge list of [open source typefaces](https://www.are.na/frederic-brodbeck/open-source-typefaces), although on a proprietary platform.)

cc @davelab6 @cameralibre @aiscarvalho @xuv @raphaelbastide @latsami @jdittrich @evalica @n8willis @qwazix and others what do you think?

**Comment by qwazix** [comment]
I know I am guilty of not updating this repository and yeah, probably we shouldn't keep it. I do love the convenience of just cloning one repo in the fonts folder and having all my fonts available so I might keep the original repo mostly for personal use.

The only way this repository might have some value is if we curated these fonts for quality as not all fonts in the other resources are of sufficiently high quality and sometimes it's better to have a smaller selection rather than a sea of options.

**Comment by elioqoshi** [comment]
I agree that we should remove it as it's not constantly updated and there are many other groups which do a better job at listing free open fonts. We could have a website page instead listing those.
